lib/license_check.py

```diff
@@ -0,0 +1,16 @@
+"""
+Try to auto-attach in a GCP instance. This should only work
+if the instance has a new UA license attached to it
+"""
+import logging
+
+from uaclient.cli import setup_logging
+from uaclient.config import UAConfig
+from uaclient.jobs.license_check import gcp_auto_attach
+
+LOG = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    cfg = UAConfig()
+    setup_logging(logging.INFO, logging.DEBUG)
+    gcp_auto_attach(cfg=cfg, logger=LOG)
```

Comment:
You don't need to pass the logger here, because you are configuring the root logger. When you get a logger inside the job itself, it will follow the configuration from `setup_logging` (unless you are setting up a child logger)

Reply:
Aye, I suspected I'd end up configuring a separate logger, but honestly haven't paid attention to the logging details yet as I was waiting for #1789 to become finalized